Gopher-Industries/Team-TaskforceX
# 209-Create-React-Component-For-Filter-Chips

Summary of changes:
- Created a new component that is visually the same as the filter chips in figma
- Brought across images that were exported from figma that will likely need to be used elsewhere too.
- Import google fonts.

Testing instructions:
As this change is bringing in a new component, you will not be able to test the functionality until it has been implemented in one of the screens.
I suggest that you carefully review the code change and images being brought in.
If you really want to test the functionality you can see how this works in 146-Customer-POV]-Onboarding-Journey-Meal-Planning-Frontend pr.
Please note this should be merged before 146-Customer-POV]-Onboarding-Journey-Meal-Planning-Frontend
